app/src/org/commcare/fragments/personalId/PersonalIdPhoneFragment.java

```diff
@@ -373,20 +378,45 @@ public void onFailure(@androidx.annotation.Nullable PersonalIdOrConnectApiErrorC
                         break;
 
                     case FORBIDDEN_ERROR:
```

Comment:
would this ever happen now given you are using a new constant for it called `INTEGRITY_ERROR` ?